utilsplot.py

```diff
@@ -97,7 +97,51 @@ def plot_freq_subplot(plt, w, direction, name, color, figure_num):
         plt.title(name)
         plt.semilogx(w, direction[i, :], color)
 
-
+        
+def complexplane(*args, color=True, marker='o', msize=5):
```

Comment:
This syntax is invalid in Python 2. Perhaps just pass the arguments as a list? So instead of using `complexplane((1+2j, 1-2j), (1, 2))` you would do `complexplane([(1+2j, 1-2j), (1, 2)])`. Boils down to removing the star and adding an extra set of brackets when calling.